lite/requirements.txt

```diff
@@ -1,2 +1,3 @@
-jupyterlite==0.6.3
-jupyterlite-xeus==4.0.5
+jupyterlite==0.6.4
```

Comment:
I'd like to take up the task of updating to `jupyterlite==0.7.0a2` in a separate PR, as `jupyterlite-xeus` will also need a new release that undoes the `jupyterlite<0.7` pin.